app/Http/Transformers/ActionlogsTransformer.php

```diff
@@ -31,49 +39,9 @@ public function transformActionlog (Actionlog $actionlog, $settings = null)
             $meta_array = json_decode($actionlog->log_meta);
 
             if ($meta_array) {
-                foreach ($meta_array as $key => $value) {
-                    foreach ($value as $meta_key => $meta_value) {
-
-                        if (is_array($meta_value)) {
-                            foreach ($meta_value as $meta_value_key => $meta_value_value) {
-                                if (is_scalar($meta_value_value)) {
-                                    $clean_meta[$key][$meta_value_key] = e($meta_value_value);
-                                } else {
-                                    $clean_meta[$key][$meta_value_key] = 'invalid scalar: '.print_r($meta_value_value, true);
-                                }
-                            }
-                        } else {
-
```

Comment:
My only concern here is that every time we pull something that starts with “this is weird, but…”, it ends up biting us for older data, newer data, etc. I’m not saying no here, just that we have to be really, really careful. I don’t recall when we added log_meta, but it’s been around a while.